content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: 'Environment variables reference'
+metaTitle: 'Prisma environment variables reference'
+metaDescription: 'This page gives an overview of all environment variables avalaible for use.'
+tocDepth: 2
+---
+
+## Overview
+
+This document describes different environment variables and their use cases.
+
+See [Connection URLs](../database-reference/connection-urls) for a breakdown of each database connector.
```

Comment:
I don't think this sentence is correct here?

Reply:
Ahh good catch, forgot to remove it